parachains/runtimes/contracts/contracts-rococo/src/contracts.rs

```diff
@@ -12,6 +12,7 @@ use pallet_contracts::{
 	weights::{SubstrateWeight, WeightInfo},
 	Config, DefaultAddressGenerator, DefaultContractAccessWeight, Frame, Schedule,
 };
+use pallet_contracts_xcm::{Config as XcmConfig, Extension as XcmExtension};
```

Comment:
```suggestion
use pallet_contracts_xcm::{Config as ContractsXcmConfig, Extension as XcmExtension};
```

Don't think we want to conflate this with being the "general" XCM config

Reply:
We are already in the contracts module. I think there is even a clippy lint that annoys you when your type is prefixed with the module name.